packages/@ionic/cli/src/commands/deploy/manifest.ts

```diff
@@ -22,19 +31,27 @@ export class DeployManifestCommand extends DeployCoreCommand {
   async getMetadata(): Promise<CommandMetadata> {
     return {
       name: 'manifest',
-      type: 'project',
+      type: 'global',
       summary: 'Generates a manifest file for the deploy service from a built app directory',
       groups: [MetadataGroup.PAID],
     };
   }
 
   async run(): Promise<void> {
-    if (!this.project) {
+    const capacitorConfig = await this.getCapacitorConfig();
+    if (!this.project && !capacitorConfig) {
       throw new FatalException(`Cannot run ${input('ionic deploy manifest')} outside a project directory.`);
     }
-    await this.requireNativeIntegration();
 
-    const buildDir = await this.project.getDistDir();
+    let distDir: string;
+    if (this.project) {
+      await this.requireNativeIntegration();
+      distDir = await this.project.getDistDir();
+    } else {
+      distDir = capacitorConfig!.webDir ? capacitorConfig!.webDir : 'www';
```

Comment:
Even though we attempt to use the `cap config --json` command which should always return a `webDir` property, there's a chance that a user has not updated to Capacitor 3 so we still need to provide a fallback.